src/app/pages/BlockDetailPage/EventsCard.tsx

```diff
@@ -34,11 +29,7 @@ const EventsList: FC<{ scope: SearchScope; blockHeight: number; filterMode: Even
 
   const { isLoading, isError, data } = eventsQuery
 
-  const events = data?.data.events.filter(
-    event =>
-      !event.tx_hash && // TODO: remove filtering here if it's implemented using the query parameters
-      (filterMode === EventFilterMode.All || event.type !== RuntimeEventType.accountstransfer),
-  )
+  const events = data?.data.events.filter(event => !event.tx_hash)
```

Comment:
We want to show all events

Reply:
This is on the block page. Removing this filter will also includes events that belong to a tx. Is that what we want?

Reply:
yes

Reply:
OK, removed all filtering from there.